esmvaltool/preprocessor/__init__.py

```diff
@@ -202,12 +202,16 @@ def preprocess(items, step, **settings):
     if itype.endswith('s'):
         result.append(_run_preproc_function(function, items, settings))
     else:
-        for item in items:
-            result.append(_run_preproc_function(function, item, settings))
+        try:
+            for item in items:
+                result.append(_run_preproc_function(function, item, settings))
+        except TypeError:
```

Comment:
When does this exception happen and why should it be ignored?

Reply:
Is just code to add a breakpoint that I forgot to remove